scripts/setup_iso.py

```diff
@@ -21,14 +22,30 @@
 @atexit.register
 def cleanup_tmp():
     if tmpdir != '':
-        subprocess.check_call(['umount', tmpdir])
+        subprocess.check_call(['umount', '-l', tmpdir])
         os.rmdir(tmpdir)
 
 def get_iso_info(fname):
     # Access ISO to determine OS type from supported list
     label1=subprocess.Popen(['isoinfo', '-i', fname, '-d'], stdout=subprocess.PIPE)
     return label1.stdout.read()
 
+def get_setup_exe_name_version(srcdir):
```

Comment:
Good suggestion Felix. I've thought of the same thing while doing coding, but I finally decided not to use windows to replace setup_exe because I'm not 100% sure the iso that contains setup.exe is equivalent to windows, although it is the case now. So while determining the OS name, on seeing setup.exe, I still used ‘get_setup_exe_name_version’ function to interpret until I get the  Product Name "Windows". It's kind of conservative, but just to be caustious